flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java

```diff
@@ -459,7 +459,8 @@ public static String[] mergeListsToArray(List<String> base, List<String> append)
             ConfigOptions.key("parallelism.default")
                     .intType()
                     .defaultValue(1)
-                    .withDescription("Default parallelism for jobs.");
+                    .withDescription(
```

Comment:
https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=66528&view=logs&j=fc5181b0-e452-5c8f-68de-1097947f6483&s=ae4f8708-9994-57d3-c2d7-b892156e7812&t=995c650b-6573-581c-9ce6-7ad4cc038461&l=31682

From the above link, we could check why CI is failed: `Documentation is outdated, please regenerate it according to the instructions in flink-docs/README.md.`

After updating the description, the document should be updated together via running `mvn package -Dgenerate-config-docs -pl flink-docs -am -nsu -DskipTests -Pskip-webui-build`.

Reply:
Thank you for the reminder.